docs/nlu/entity-extraction.rst

```diff
@@ -98,11 +98,82 @@ Custom Entities
 Almost every chatbot and voice app will have some custom entities.
 A restaurant assistant should understand ``chinese`` as a cuisine,
 but to a language-learning assistant it would mean something very different.
-The ``CRFEntityExtractor`` component can learn custom entities in any language, given
+The ``CRFEntityExtractor`` and the ``DIETClassifier`` component can learn custom entities in any language, given
 some training data.
 See :ref:`training-data-format` for details on how to include entities in your training data.
 
 
+.. _composite-entities:
+
+Composite Entities
+^^^^^^^^^^^^^^^^^^
+
+Assigning custom entity labels to words, allow you to define certain concepts in the data.
+For example, we can define what a `city` is:
+
+.. code-block:: none
+
+    I want to fly from [Berlin](city) to [San Francisco](city).
+
+However, sometimes you want to specify entities even further.
+Let's assume we want to build an assistant that should book a flight for us.
+The assistant needs to know which of the two cities in the example above is the departure city and which is the
+destination city.
+``Berlin`` and ``San Francisco`` are still cities, but they play a different role in our example.
+Composite entities allow you to assign a role and/or a group label next to the entity label.
+
+.. code-block:: none
+
+    I want to fly from [Berlin]{"entity": "city", "role": "departure"} to [San Francisco]{"entity": "city", "role": "destination"}.
+
+The group label might be useful in the following situation:
+
+.. code-block:: none
+
+    Give me a [small]{"entity": "size", "group": "1"} pizza with [mushrooms]{"entity": "topping", "group": "1"} and
+    a [large]{"entity": "size", "group": "2"} [pepperoni]{"entity": "topping", "group": "2"}
```

Comment:
Is the "group" here to reference which topping goes with which pizza? it's not immediately obvious to me.

Reply:
Yes, will add a sentence.